openjdk/openjdk.build

```diff
@@ -8,6 +8,15 @@
         <property name="pathsep" value=";" />
     </if>
 
+    <target name="all.netstandard">
```

Comment:
Actually, I believe we can stop using nant and use something new build script automation tools like [cakebuild](cakebuild.net) or [nuke](https://nuke.build/). IMO, this will be more readable and supportable.